mantidproject/mantid
# Upgrade UI and preprocessing for HB3A

Move experimental data downloading via http server from main UI to a dialogue window.

Improve the usability of pre-processing.

**To test:**

1. check UI;
2. pre-process some HB3A scans;
3. close Mantid and re-launch and try to merge the scans that are previously processed.



Fixes #20925 . 

**Release Notes** 

Release note has been added to docs/source/release/v3.12.0/diffraction.rst

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
